jwst/conftest.py

```diff
@@ -108,3 +112,17 @@ def pytest_runtest_logstart(self, nodeid, location):
             yield
             if self.desc:
                     self.terminal_reporter.write(f'\n{self.desc} ')
+
+
+def get_crds_context():
```

Comment:
There's a [method](https://github.com/spacetelescope/stpipe/blob/0.1.0/src/stpipe/crds_client.py#L179) in `stpipe.crds_client` that returns this value.  It uses `crds.heavy_client.get_processing_mode` which is a good choice because it returns the actual context that the crds client is using.

Reply:
Thanks!  Perfect.  Changed the code.